advanced/creating-a-skin.md

```diff
@@ -4,48 +4,56 @@ title: Create a skin for TinyMCE
 
-## Modifying the less files
+### Creating a skin
 
-Most CSS elements can be modified by altering the `variables.less` file. This file contains variables such as:
-* font size
-* font family
-* colors
-* borders
+1. Begin by duplicating the `default` folder located in `src/less/skins/ui/` and rename it to the name of your skin.
+2. Open the file `src/less/theme/globals/global-variables.less` and **copy** the variables you like to change to your `skin.less` file in the folder you duplicated in the previous step. Change the values. The variables you put in `skin.less` will override the default values.
+3. For more detailed customizations, review the variables in each component, such as `src/less/theme/components/toolbar-button.less` and copy the ones you want to change to `skin.less`.
+4. To style interface elements in the content, such as selection color, drag handles, table of contents, bookmarks etc, copy variables to `src/less/skins/ui/<skin-name>/content.less` or `content.inline.less` depending on if you are using the inline mode or not.
+5. Preview your skin by building it using the `gulp` command in your terminal.
 
-Modify additional files to create radically different skins. 
+### Creating a content css
 
-> Note: The skin **only** changes the visual presentation of the UI and **not** the placement of elements. Placement of elements is done by the TinyMCE UI framework. This framework makes it possible to do complex UI layouts on all browsers without touching any CSS when plugins are created.
+To update the appearance of the content within the editor, such as headings, quotes, lists etc you create a content css. These are located in `src/less/skin/content/`
 
-## Modifying the icons
+1. Create a folder in `src/less/skins/content/` and create a `content.less` file in it. Alternatively you can duplicate any of the existing content skins.
+2. Add the relevant element selectors for your particular use case such as `h1` to `h6`, `a`, `blockquote`, `code`, `table`... 
+
+### Moving your skin into TinyMCE
 
-The icons shipped with the default skin are created by the [IcoMoon](http://icomoon.io/) project and are embedded as web fonts. This makes them *retina ready* and easy to skin.
+1. Simply copy your skin and/or content skin from `build/skins/ to the corresponding folders in your TinyMCE folder.
+2. Update your init function with the [skin](https://www.tiny.cloud/docs-beta/configure/editor-appearance/#skin) option and/or the [content_css](https://www.tiny.cloud/docs-beta/configure/content-appearance/#content_css) option.
 
-Import the existing TinyMCE icons by uploading the "icomoon.dev.svg" file and then adding and/or changing the icons that are provided. Alter the `icons.less` file to add more icons.
+## Modifying the icons
 
-> Note: There are a few images that are used as placeholders within the contents of the editor.
+Documentation on how to change the icons will be added here shortly.
```

Comment:
This should probably link to other pages, rather than including it here, since I think the explanation might be quite long?

Reply:
We can still keep the notice here as the v4 docs had icons documentation on this page...